nano/nano_node/entry.cpp

```diff
@@ -266,26 +266,12 @@ int main (int argc, char * const * argv)
 			nano::inactive_node node (data_path);
```

Comment:
I guess inactive_node needs special launch flag to not perform reps weights calculation each time. Expect for debug_dump_representatives

Reply:
I've added a flag in the `inactive_node` constructor as that seemed easiest, also added it to the `node_flags` which is constructed in the `inactive_node` constructor as well. Possibly `inactive_node` should take `node_flags` as well, but each one CLI call would then have to set it to false. Something to think about in a future PR maybe :)